compiler/modules/bir/verify.bal

```diff
@@ -151,10 +151,10 @@ function verifyCall(VerifyContext vc, CallInsn insn) returns err:Semantic? {
     if nSuppliedArgs != nExpectedArgs {
         string name = vc.symbolToString(func.symbol);
         if nSuppliedArgs < nExpectedArgs {
-            return vc.err(`too few arguments for call to function ${name}`);
+            return vc.err(`too few arguments for call to function ${name}`, insn.pos);
         }
         else {
-            return vc.err(`too many arguments for call to function ${name}`);
+            return vc.err(`too many arguments for call to function ${name}`, insn.pos);
```

Comment:
The position needs to be passed through to and used in verifyOperandType, both here and elsewhere: operandTypeErr needs a Position? argument (which will change to a Position)

Reply:
fixed